README.md

```diff
@@ -87,12 +87,11 @@ There is an official image, built from source at: https://hub.docker.com/r/nofis
 
 ### Install from source
 
- - `wget https://github.com/HelloZeroNet/ZeroNet/archive/py3/ZeroNet-py3.tar.gz`
- - `tar xvpfz ZeroNet-py3.tar.gz`
- - `cd ZeroNet-py3`
- - `sudo apt-get update`
- - `sudo apt-get install python3-pip`
- - `sudo python3 -m pip install -r requirements.txt`
+ - `sudo su`
+ - `apt update && apt full-upgrade && apt install git python3-pip`
+ - `git clone https://github.com/hellozeronet/zeronet`
+ - `cd zeronet`
+ - `pip3 install --upgrade --no-cache-dir -r requirements.txt`
```

Comment:
Please at the very least replace `pip3` with `python3 -m pip` because `pip3` might refer to PyPy pip version or can not exist at all (e.g. on some Windows builds).

Reply:
`--no-cache-dir` disables, uh, cache which is inefficient. This is also an ugly solution to the "old gevent" problem when a clean one exists.